package.json

```diff
@@ -36,7 +36,7 @@
     "test-headless": "mochify --no-detect-globals --recursive -R dot --grep WebWorker --invert --plugin [ proxyquire-universal ] \"test/**/*-test.js\"",
     "test-coverage": "nyc npm run test-headless -- --transform [ babelify --ignore [ test ] --plugins [ babel-plugin-istanbul ] ]",
     "test-cloud": "npm run test-headless -- --wd",
-    "test-webworker": "mochify --no-detect-globals --https-server 8080 --no-request-interception test/webworker/webworker-support-assessment.js",
+    "test-webworker": "mochify --no-detect-globals --https-server 8080 --no-request-interception test/webworker/webworker-support-assessment-test.js",
```

Comment:
Wasn't this named without `-test` on purpose to exclude it from the other test runs?

Reply:
Ooh, I don't remember exactly, but I think you are right. We should not run it as part of a Mocha suite, just when explicitly triggered, and if running Mocha recursively, then this might be included.

Reply:
You're right! Thank you for spotting this.

I've updated the affected files to use inline configuration of ESLint, so they will not trip up with the new configuration and also won't be matched and run with the other tests.

🙏

Reply:
That replaced the silly commit with two new ones, at the top of the PR